response_operations_ui/controllers/message_controllers.py

```diff
@@ -11,10 +11,10 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def get_message_list(params):
```

Comment:
Will we ever need to use get_message_list again? (I'm guessing no)

Reply:
Not as far as I know, the only use of it became the threads views.